src/scenes/app/stores/index.js

```diff
@@ -0,0 +1,43 @@
+import { observable, action } from 'mobx';
+import sampleData from '../../../helpers/dataSourceSample';
+import strings from '../../../localization/en/strings';
+
+class ObservableTodoStore {
+  @observable items = sampleData;
+  @observable error = '';
+
+  @action
```

Comment:
If would be good to enable the `strict` mode in mobx, just to be sure that you only change the state when in an action